pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -157,6 +184,42 @@ func applyServiceDefinition(ctx context.Context, cli provider.Client, serviceDef
 	return service, nil
 }
 
+func applyStandaloneTask(
+	ctx context.Context,
+	cli provider.Client,
+	clusterDefinition types.Cluster,
+	awsVpcConfiguration types.AwsVpcConfiguration,
+	taskDefinition types.TaskDefinition,
+) (*ecs.RunTaskOutput, error) {
+	output, err := cli.RunTask(ctx, clusterDefinition, awsVpcConfiguration, taskDefinition)
+	if err != nil {
+		return nil, fmt.Errorf("unable to run ECS task %s: %v", *taskDefinition.TaskDefinitionArn, err)
+	}
+	return output, nil
+}
+
+func runStandaloneTask(
+	ctx context.Context,
+	in *executor.Input,
+	cloudProviderName string,
+	cloudProviderCfg *config.PlatformProviderECSConfig,
+	clusterDefinition types.Cluster,
+	awsVpcConfiguration types.AwsVpcConfiguration,
+	taskDefinition types.TaskDefinition,
```

Comment:
we're customary to pass logs at the end of args, so modify them as this.
```suggestion
func runStandaloneTask(
	ctx context.Context,
	cloudProviderName string,
	cloudProviderCfg *config.PlatformProviderECSConfig,
	clusterDefinition types.Cluster,
	awsVpcConfiguration types.AwsVpcConfiguration,
	taskDefinition types.TaskDefinition,
	lp executor.LogPersister,
```

Reply:
I am not very good at English yet, so I am sorry if I did not understand your intentions.

Since `in *executor.Input` is used in `client, err := provider.DefaultRegistry().Client(cloudProviderName, cloudProviderCfg, in.Logger)`, we need to pass `in *executor.Input`.
Also, `in *executor.Input` has `LogPersister` and `*zap.Logger`.

https://github.com/pipe-cd/pipecd/pull/4084/files#diff-0ced6421324def8505cb6dadaa49f469d0855ef22437ba6d8ad7f5279a61b2e4R210

Reply:
oh, right. thanks.